src/transformers/modeling_tf_utils.py

```diff
@@ -122,7 +135,9 @@ def compute_loss(self, labels, logits):
         loss_fn = tf.keras.losses.SparseCategoricalCrossentropy(
             from_logits=True, reduction=tf.keras.losses.Reduction.NONE
         )
-        active_loss = tf.reshape(labels, (-1,)) != -1
```

Comment:
@jplu @sgugger @LysandreJik - In PyTorch we use -100 and not -1 to mask tokens for the loss. Should we do the same here? 
Would slightly break backward compatibility since -1 was already used for token classification - but not sure how many people already trained on token classification.